slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -340,6 +354,37 @@ void SLAMServiceImpl::BackupLatestMap() {
     latest_pointcloud_map = std::move(pointcloud_map_tmp);
 }
 
+// If using the LOCALIZING action mode, cache a copy of the map before
+// beginning to process data. If cartographer fails to do this,
+// terminate the program
+bool SLAMServiceImpl::CacheMapInLocalizationMode() {
```

Comment:
[minor]
```suggestion
void SLAMServiceImpl::CacheMapInLocalizationMode() {
```

Reply:
was already making changes so I added this to my commit